### ISSUE TITLE
Examples broken in CSS Painting API Guide

### ISSUE THREAD

**Issue by darabos** [mdn/content#23755]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/CSS_Painting_API/Guide

### What information was incorrect, unhelpful, or incomplete?

The example is not working due to CORS. The JavaScript console says:

> Access to script at 'https://mdn.github.io/houdini-examples/cssPaint/intro/01partOne/header-highlight.js' from origin 'https://yari-demos.prod.mdn.mozit.cloud' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.

### What did you expect to see?

Thanks! I love MDN!

**Comment by wbamberg** [comment]
Thanks for filing! I think this is a duplicate of https://github.com/mdn/content/issues/23394. 

@fiji-flo , this seems to be a case where we can't use live samples because of CORS, and would be forced to use GitHub pages for?
